src/Expand/Expand.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, StyleSheet, TouchableOpacity } from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+
+export default class Expand extends Component {
+  static propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+    children: PropTypes.node,
+    defaultExpanded: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    expanded: false,
+  }
+  constructor(props) {
+    super(props)
+    this.state = { expanded: props.defaultExpanded }
+  }
+
+  onPress = () => {
+    this.setState(
+      { expanded: !this.state.expanded }
+    )
+  }
+
+  render() {
+    const { title, description, children } = this.props
+    const { expanded } = this.state
+    const iconName = expanded ? 'chevron-up' : 'chevron-down'
+
+    return (
+      <TouchableOpacity onPress={this.onPress} activeOpacity={1}>
+        <View style={styles.container}>
+          <View style={styles.titleContainer}>
+            <Text numberOfLines={1}>
+              {title}
```

Comment:
@densa I think @itsmepetrov meant that you should describe styles for Text too, cause we should provide a way to customize styles for this Text (color, fontSize, fontWeight, etc...)